### DISCUSSION TITLE
Thank you mantine 💘

### DISCUSSION THREAD

**Discussion by simon-lehmann** [mantinedev/mantine#3741] [Kind words]
Dear Mantine Team,
thank you for putting this library together. I have started to use and love mantine in my free time, and bringing this great library to good use in our company now. All the developers are very pleased with the development experience, the time savings for any bigger project is insane. The amount of flexibility we have with our designers and developers will result in great products. All thanks to every contributor.
Continue the good work!

**Comment by rtivital** [comment]
Thanks for kind words!
